src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars div {
+  display: flex;
+}
+
+.stars--0 div,
```

Comment:
add class, don't style div's 
`.stars__item` maybe

Reply:
>Each block should have a given number of active stars without adding any special class for an individual star. 
https://github.com/mate-academy/layout_stars

Reply:
it means not to set different classes for stars, but 1 for all needs to be created. because  styling divs that's a bad idea